src/vs/server/node/serverServices.ts

```diff
@@ -123,14 +123,19 @@ export async function setupServerServices(connectionToken: ServerConnectionToken
 	const uriIdentityService = new UriIdentityService(fileService);
 	services.set(IUriIdentityService, uriIdentityService);
 
+	// Configuration
+	const configurationService = new ConfigurationService(environmentService.machineSettingsResource, fileService, new NullPolicyService(), logService);
+	services.set(IConfigurationService, configurationService);
+
 	// User Data Profiles
 	const userDataProfilesService = new ServerUserDataProfilesService(uriIdentityService, environmentService, fileService, logService);
 	services.set(IUserDataProfilesService, userDataProfilesService);
 
-	// Configuration
-	const configurationService = new ConfigurationService(environmentService.machineSettingsResource, fileService, new NullPolicyService(), logService);
-	services.set(IConfigurationService, configurationService);
-	await configurationService.initialize();
+	// Initialize
+	await Promise.all([
```

Comment:
Should this also include the `await getMachineId()` below to run everything in parallel?

Reply:
Done